test/api/v3/unit/libs/cron/cron.habits.test.js

```diff
@@ -0,0 +1,308 @@
+import * as Tasks from '../../../../../../website/server/models/task';
+import { cron } from '../../../../../../website/server/libs/cron';
+import analytics from '../../../../../../website/server/libs/analyticsService';
+import { model as User } from '../../../../../../website/server/models/user';
+
+describe('habits', () => {
+  let user;
+  let tasksByType = {habits: [], dailys: [], todos: [], rewards: []};
+  let daysMissed = 0;
+
+  beforeEach(() => {
+    user = new User({
+      auth: {
+        local: {
+          username: 'username',
+          lowerCaseUsername: 'username',
+          email: 'email@email.email',
```

Comment:
This is pretty minor and picky so it's just a suggestion: perhaps we should use example.com rather than email.email. ".email" is a gTLD so it's not impossible that that email address would actually exist and the owner might be peevish about it appearing in github.